VKappProject/VKappProject/Controllers/AvailableGroupsTableViewController.swift

```diff
@@ -1,27 +1,18 @@
 // AvailableGroupsTableViewController.swift
 // Copyright © RoadMap. All rights reserved.
 
+import Alamofire
 import UIKit
 
-/// Экран доступных групп
+/// Доступные группы
 final class AvailableGroupsTableViewController: UITableViewController {
     // MARK: - Private IBOutlets
 
     @IBOutlet private var groupsSearchBar: UISearchBar!
 
-    // MARK: - Public Properties
-
-    var closureForGroup: ((Group) -> Void)?
-
     // MARK: - Private Properties
 
-    private var availableGroups = Group.groups {
-        didSet {
-            tableView.reloadData()
-        }
-    }
-
-    private lazy var searchResult: [Group] = []
+    private var searchResult: [GroupItem] = []
 
     private var searchResultIsEmpty: Bool {
```

Comment:
```suggestion
    private var isSearchResultEmpty: Bool {
```
тут вопросительная форма для бинарного ответа а не утвердительная что есть пустой результат